src/libraries/System.Security.Cryptography.Cose/tests/CoseTestHelpers.cs

```diff
@@ -420,13 +568,39 @@ private static RSA CreateRSA(bool includePrivateKey)
             return RSA.Create(rsaParameters);
         }
 
+        private static MLDsa CreateMLDsa(MLDsaAlgorithm algorithm, bool includePrivateKey)
+        {
+            MLDsaNistTestCase nistKey = MLDsaTestsData.GetPassingNistTestCase(algorithm);
+
+            if (includePrivateKey)
+            {
+                return MLDsa.ImportMLDsaSecretKey(algorithm, nistKey.SecretKey);
+            }
+            else
+            {
+                return MLDsa.ImportMLDsaPublicKey(algorithm, nistKey.PublicKey);
+            }
+        }
+
         internal static bool AlgorithmNeedsHashAlgorithm(CoseAlgorithm algorithm)
             => algorithm is
                 CoseAlgorithm.ES256 or CoseAlgorithm.ES384 or CoseAlgorithm.ES512 or
                 CoseAlgorithm.PS256 or CoseAlgorithm.PS384 or CoseAlgorithm.PS512 or
                 CoseAlgorithm.RS256 or CoseAlgorithm.RS384 or CoseAlgorithm.RS512;
 
-        internal static (T Key, HashAlgorithmName Hash, RSASignaturePadding? Padding) GetKeyHashPaddingTriplet<T>(CoseAlgorithm algorithm, bool useNonPrivateKey = false)
+        internal static bool AlgorithmIsSupported(CoseAlgorithm algorithm)
+        {
+            switch (algorithm)
+            {
+                case CoseAlgorithm.MLDsa44:
+                case CoseAlgorithm.MLDsa65:
+                case CoseAlgorithm.MLDsa87:
+                    return MLDsa.IsSupported;
+                default: return true;
```

Comment:
Nit (not worth respinning for, but if there's any other edit, please fix).

```suggestion
                default:
                    return true;
```